src/app.js

```diff
@@ -29,34 +35,42 @@ export default (
   ): mixed => {
     if (request.method === 'OPTIONS') {
       response.set({
-        'Access-Control-Allow-Headers':
-          'X-Requested-With, Content-Type, Accept, Authorization',
+        'Access-Control-Allow-Headers': 'X-Requested-With, Content-Type, Accept, Authorization',
```

Comment:
This is more than 80 chars on a line. We purposely put the value of this on its own line.

@AntonPuko - can we add `prettier` to this so we don't run into formatting issues?

It should be part of the commit hooks.